src/core/Akka/IO/SocketEventArgsPool.cs

```diff
@@ -21,48 +22,81 @@ public interface ISocketEventArgsPool
     {
         SocketAsyncEventArgs Acquire(IActorRef actor);
         void Release(SocketAsyncEventArgs e);
+        
+        BufferPoolInfo BufferPoolInfo { get; }
     }
 
     internal class PreallocatedSocketEventAgrsPool : ISocketEventArgsPool
     {
+        private readonly IBufferPool _bufferPool;
```

Comment:
Buffer pool belongs to the socket event pool now, no one should touch this. ever.

Reply:
Add a comment saying this and explaining why